### DISCUSSION TITLE
Vstream

### DISCUSSION THREAD

**Discussion by Oruspo09** [Kodi-vStream/venom-xbmc-addons#3931] [Support & aide]
Bonjour depuis 2 jours vstream ne fonctionne plus tout les sites n'affiche aucun film pourtant dns a jours tout à jours ?
Avez vous le même problème merci ?

**Comment by tomtom3986** [comment]
bonjour , je suis pas expert mais moi aussi plus de resultat
avec  1 clé hdmi xiaomi
1 mibox3 xiaomi
et 1 pc sous w10 
meme symptome
au demarrage de kodi (ou vstream)
j'ai eut tous un tas de ligne (du code? ces ligne etaient partiellement comprehensible ) qui se sont affiché sur la droite de l'ecran
mes 3 machine sont touché
certaine sont  en dns google et d'autre en dns cloudfare

**Reply by tomtom3986** [reply]
aprés avoir desinstallé et reinstallé  vstream tous est rentré en ordre pour le moment.

**Comment by sizanic** [comment]
Bonjour Oruspo9,

Normalement, au moins le site Wiflix devrait te proposer des liens fonctionnelles.

Pour autant, une mise à jour de vStream arrive cette semaine, et devrait rendre de nouveau tous les sites fonctionnelles.

**Comment by TmpName** [comment]
Hello, quand vous dites "TOUT" les sites, donnez 3 / 4 exemples qu'on puisse tester de notre coté avec les memes pour savoir d'ou vient le probleme, ca evite le "je viens de tester chez ca marche sans probleme", car il y a plusieurs dizaines de sites

Regardez aussi vos logs, si la panne est generale, il y a de forte chance qu'un gros probleme  y soit indiqué.

**Comment by Oruspo09** [comment]
Bonjour non je confirme aucun site ne fonctionne et n'affiche de film même wiflix qui d'habitude es au top en espérant que la mise à jour arrive vite merci

Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by TmpName** [comment]
Ok donc j'ai eu le meme bug ^^.
Le fichier site.json qui contenait "" du coup, plus aucun site ne fonctionnait.
Par contre je sais pas ce qui c'est passé.

https://github.com/Kodi-vStream/venom-xbmc-addons/blob/Beta/plugin.video.vstream/resources/sites.json

A voir si ca venait pas de github.

**Comment by Oruspo09** [comment]
Du coup on attend il y aura une mise à jours avec les corrections ?

**Comment by sizanic** [comment]
> Le fichier site.json qui contenait "" du coup, plus aucun site ne fonctionnait.
Par contre je sais pas ce qui c'est passé.

Le fichier n'était pas vide mais il était mal formaté.
Il contenait une virgule derrière le dernier élément, erreur de ma part.
https://github.com/Kodi-vStream/venom-xbmc-addons/commit/1024a61c7e5cfe8468ee21ba09fbb3d6603e696d#diff-f1f6505b68d3cb6d71eb8ea5e4d585e36b1ebc7805facfd54cb278464ab5abc5

Problème corrigé.


Par contre, le symptome n'est pas l'absence de films dans les sources 🤔 
Mais le symptome devait être l'absence de sources.
=> voire même l'impossibilité de naviguer dans les menus.



> Du coup on attend il y aura une mise à jours avec les corrections ?

non, si c'est bien le souci, le fichier a été corrigé il y a deux jours, donc même sans mise à jour il devrait déjà être déployé sur ton installation. Et ça va refonctionner sans rien faire.

**Comment by TmpName** [comment]
Ben chez moi il contenait juste "", je pense que le code python qui recuperait le json a planté a cause de la faute de frappe et retournait une chaine vide.
Et toutes les sources etaient visible (c'est juste des menus), a part que aucun lien ne marchaient (car les urls etaient bancales).

Mais c'etait visible, dans chaque site quand ca deconnait, il y a avait un message d'erreur qui expliquait que les url etaient mauvaises, et je suis sur que c'etait ecrit en clair dans les logs.

**Reply by sizanic** [reply]
ok je vois, c'est parce que j'utilise la Beta que le comportement est différent.

**Reply by TmpName** [reply]
Yep suis sur l'officielle (enfin presque, vu que apparement j'ai des modifs que seul moi ai et dont je ne me souvient meme plus)

**Comment by Oruspo09** [comment]
Slt je comprends pas tu veux faire quoi

Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________
